tests/b908.py

```diff
@@ -0,0 +1,28 @@
+import unittest
```

Comment:
Should also add some tests with only one statement inside to make sure it does not trigger for those.